.github/workflows/ai-codereview.yml

```diff
@@ -0,0 +1,61 @@
+name: AI code review
+on:
+  pull_request:
+    branches: [ "main" ]
```

Comment:
should we execute the jobs on each and every PR? i'd suggest to skip some paths/extensions (e.g. `**.md`, `config/**`, `hack/**`, etc.)